app/controllers/apartments_controller.rb

```diff
@@ -3,4 +3,19 @@ def index
         apartments = Apartment.all
         render json: apartments
     end
+
+    def create
+        apartment = Apartment.create(apartment_params)
+        # if apartment.valid?
```

Comment:
if you are not going to use the conditions for the create endpoint, please remove commented code on lines 9, 12-14.